client/scss/settings/_variables.scss

```diff
@@ -102,45 +102,8 @@ $system-color-button-text: ButtonText;
 // Our fonts are based off of a list of system fallbacks to ensure
 // that the most 'up-to-date' and available system font is used and rendered consistently as possible across browsers.
 
-$font-sans:
-  // iOS Safari, macOS Safari, macOS Firefox
-  -apple-system,
-  // macOS Chrome
-  BlinkMacSystemFont,
-  // Windows - for all browsers on Windows 7+ (putting Segoe UI before system-ui ensures Segoe UI will be rendered for different languages)
-  'Segoe UI',
-  system-ui,
-  // Targets Android and newer Chrome OS'. (If Roboto is installed on your windows computer Segoe UI will take precedence.)
-  Roboto,
-  // A common fallback font for older macOS'
-  'Helvetica Neue',
-  // Very old Windows versions (special shout-out to whoever is using windows 95)
-  Arial,
-  // A last resort if all else fails, just give us something without serifs :)
-  sans-serif,
-  // All the emojis 👋🙂
-  'Apple Color Emoji',
-  'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
```

Comment:
I think it would be nice for us to move those comments over to `typography.js` as this took quite a bit of research.

Reply:
done